plugins/UsersManager/templates/index.twig

```diff
@@ -178,6 +178,7 @@
                 {% if showLastSeen is defined and showLastSeen %}
                 <th>{{ 'UsersManager_LastSeen'|translate }}</th>
                 {% endif %}
+                <th>token_auth</th>
```

Comment:
Can we maybe remove this column and instead use an icon in front of the shown token? The button is quite big in there and rarely needed. eg 

`<span class="icon-reload"></span> token...` We could maybe also find a new icon and add it